spec/spec_helper.rb

```diff
@@ -9,6 +9,7 @@
 $LOAD_PATH.unshift(File.dirname(__FILE__))
 $LOAD_PATH.unshift(File.join(File.dirname(__FILE__), '..', 'lib'))
 require 'active_hash'
+require 'active_record' unless SKIP_ACTIVE_RECORD
```

Comment:
@kbrock Fixed. Could you review and merge this pull request if it doesn't have any problems?